http_service/bugbug_http/app.py

```diff
@@ -190,8 +195,22 @@ def schedule_job(
     )
 
 
-def schedule_bug_classification(model_name: str, bug_ids: Sequence[int]) -> None:
-    """Schedule the classification of a bug_id list"""
+def prepare_queue_job(
+    job: JobInfo, job_id: Optional[str] = None, timeout: Optional[int] = None
+) -> Queue:
+    job_id = schedule_init(job, job_id)
+    return Queue.prepare_data(
+        job.func,
+        args=job.args,
+        job_id=job_id,
+        timeout=timeout,
+        ttl=QUEUE_TIMEOUT,
+        failure_ttl=FAILURE_TTL,
+    )
+
+
+def create_bug_classification_jobs(model_name: str, bug_ids: Sequence[int]):
```

Comment:
Can you specify the return type of the function?

Reply:
added return type